Fix code examples in doc for running Elastic Agent unprivileged (#4070)
elastic/docs-content: reference/fleet/elastic-agent-unprivileged.md

In certain cases you may want to install {{agent}} in `unprivileged` mode, with the agent running as a pre-existing user or as part of a pre-existing group. For example, on a Windows system you may have a service account in Active Directory and you’d like {{agent}} to run under that account.

::::{note}
On Windows, the `--password` parameter is required when specifying a custom user account.

On Linux and macOS, the `--user` and `--group` parameters are optional:
* If you omit `--user`, {{agent}} uses (or creates) the default unprivileged user (`elastic-agent-user`).

> * If you specify only `--group`, the agent runs unprivileged with the requested group using the default user.

Reviewer: ```suggestion
* If you specify only `--group`, the agent runs unprivileged in the requested group using the default user.
```

:::::{tab-set}
:group: os

::::{tab-item} Linux/macOS
:sync: linux

To install with a specific user:

```shell
sudo elastic-agent install --unprivileged --user="username"
```

To install with a specific group:

```shell
sudo elastic-agent install --unprivileged --group="groupname"
```

To install with both a specific user and group:

```shell
sudo elastic-agent install --unprivileged --user="username" --group="groupname"
```

::::

::::{tab-item} Windows
:sync: windows

On Windows, the `--password` parameter is required when specifying a custom user account.

> 

Reviewer: ```suggestion
```

In certain cases you may want to install {{agent}} in `unprivileged` mode, with the agent running as a pre-existing user or as part of a pre-existing group. For example, on a Windows system you may have a service account in Active Directory and you’d like {{agent}} to run under that account.

::::{note}

> The `--password` parameter is only required on Windows.

Reviewer: ```suggestion
On Windows, the `--password parameter` is required when specifying a custom user account.
```
